pkg/collect/host_collector.go

```diff
@@ -8,6 +8,8 @@ type HostCollector interface {
 	Title() string
 	IsExcluded() (bool, error)
 	Collect(progressChan chan<- interface{}) (map[string][]byte, error)
+	RemoteCollect(progressChan chan<- interface{}) (map[string][]byte, error)
+	HasRemoted() bool
```

Comment:
What is this for? The naming has me a bit confused.

Reply:
I think i see now. I wonder if this could be renamed to something that would make it clearer what this is for.